addons/mass_mailing/tests/test_mailing_ab_testing.py

```diff
@@ -147,3 +147,21 @@ def test_ab_testing_mail_compare_version(self):
         # Record with mailing_type equals to mail should be included in compare version count, doesn't matter
         # if campign_id is same
         self.assertEqual(compare_version_record_count, 2, 'Number of records should be according to mailing type')
+
+    @mute_logger('odoo.addons.mail.models.mail_mail')
+    @users('user_marketing')
+    def test_send_minimum_one_mail_for_ab_testing(self):
+        """ Test that it should send minimum one mail(if possible) when ab_testing_pc is zero."""
```

Comment:
You could remove "when ab_testing_pc is zero." as it is not the case here.

Could replace with something like "when ab_testing_pc is too small compared to the amount of targeted records".